tensorflow/lite/kernels/mul.cc

```diff
@@ -46,6 +46,12 @@ limitations under the License.
 #include "tensorflow/lite/minimal_logging.h"
 #endif  // TFLITE_KERNEL_USE_XNNPACK
 
+// to make mul.cc work with f16 and bf16 the following line must be commented -
+// line: 701 in "tensorflow/lite/kernels/BUILD", line: 62 in
```

Comment:
what is the meaning of this comment?

Reply:
To add f16/Bf16 support,we had added Eigen/core header which was conflicting with EigenForTFLite in tensorflow/lite/kernels/internal/optimized/eigen_spatial_convolutions.h referenced by multithreaded_conv.h in conv2d.Thus we were commenting out convolution from build.But the issue is resolved at (https://github.com/MCW-Dev/tensorflow/pull/35). And if the approach is confirmed the corresponding changes will be made in mul op and div op.Also unfortunately this file is not final changed version so is it alright if i do a git commit --amend ?

Reply:
Hi Luke, the conv error has been addressed in 7ab407c